CHARTER.md

```diff
@@ -0,0 +1,73 @@
+# Working Group Definition
+
+The Community Engagement and Support Working Group exists to strengthen human connection within the DevRel Foundation by fostering collaboration, support, and engagement among community members. Our primary objectives include:
+* **Human Connection** – Creating spaces for authentic interactions, knowledge sharing, and support among DevRel professionals.
+* **Community Growth & Engagement** – Expanding DevRel Foundation awareness and increasing DevRel professionals’ and those interested in DevRel involvement through active engagement initiatives.
+* **Digital Community Management** – Organizing and managing the DevRel Foundation Discord to foster meaningful discussions, collaboration, and engagement.
+* **Event Planning & Execution** – Designing and executing DevRel Foundation events, both virtual and in-person, to facilitate collaboration, networking, and knowledge sharing.
```

Comment:
This might be a good one to work with the resource aggregation group with, since they're going to put work into an event directory and can probably provide some great insights into ideal locations / timelines. See their charter here: https://github.com/DevRel-Foundation/wg-resource-aggregation/blob/main/CHARTER.md